sleap_roots/bases.py

```diff
@@ -12,8 +12,15 @@ def get_bases(pts: np.ndarray) -> np.ndarray:
     Returns:
         Array of bases (instances, (x, y)).
     """
-    # (instances, 2)
-    base_pts = pts[:, 0]
+    # Exceptions
```

Comment:
Comments like these should be full sentences, for example: `# Check for missing data.`

Also, refer to the cases covered by this conditional as "edge case" or "special case" since `Exception` is an actual Python built in class

Reply:
Suggested changes made.